velox/connectors/hive/HiveDataSink.cpp

```diff
@@ -37,12 +37,12 @@ std::string makeUuid() {
 }
 
 std::string makePartitionDirectory(
-    const std::string& tableDirectory,
-    const std::optional<std::string>& partitionSubdirectory) {
-  if (partitionSubdirectory.has_value()) {
-    return (fs::path(tableDirectory) / partitionSubdirectory.value()).string();
+    const std::string& tableDir,
```

Comment:
Presto Java codebase discourages abbrev. Not sure about Velox.

Reply:
tableDir is common abbreviation in c++ data system code base.

Reply:
Ok. Also comment from @oerling "We hav generally preferred directory over dir." Changes in policy? I revert this change.